app/helpers/ukims_helper.rb

```diff
@@ -41,6 +41,35 @@ def explore_topic_title
     end
   end
 
+  concerning :PlannedProcessingText do
+    def commercial_processing_hint_text
+      html_data = if cut_off_validation
+                    <<-PARAGRAPH
+          <ul>
+            <li>the sale of <strong>food</strong> to end consumers in the UK</li>
+            <li><strong>construction</strong>, where the processed goods form a permanent part of a structure that is constructed and located in Northern Ireland by the importer or one subsequent entity</li>
+            <li>direct provision to the recipient of <strong>health or care services<strong> in Northern Ireland by the importer or one subsequent entity</li>
+            <li><strong>not for profit</strong> activities in Northern Ireland by the importer or one subsequent entity, where there is no subsequent sale of the processed goods</li>
+            <li>the final use of <strong>animal feed</strong> on premises located in Northern Ireland by the importer or one subsequent entity.</li>
+          </ul>
+                    PARAGRAPH
+
+                  else
+                    <<-PARAGRAPH
+          <ul>
+            <li>the sale of <strong>food</strong> to end consumers in the UK</li>
+            <li><strong>construction</strong>, where the processed goods form a permanent part of a structure that is constructed and located in NI by the importer</li>
```

Comment:
The preview PARAGRAPH uses Northern Ireland instead of NI,
should we expand NI -> Northern Ireland as well here?

Reply:
We're not really coming up with the content and this has existed for a long time. I'd be tempted to not change it given all new trades will be in the other branch